src/webgl/context-state.js

```diff
@@ -120,7 +120,7 @@ export function setParameters(gl, parameters) {
 export function withParameters(gl, parameters, func) {
   // assertWebGLContext(gl);
 
-  const {frameBuffer, nocatch = true} = parameters;
+  const {frameBuffer, readFramebuffer, nocatch = true} = parameters;
```

Comment:
I think we can skip this.
The `readFramebuffer` is only used internally by framebuffer.blit and framebuffer.readPixels. 
I do not expect that the application would need to bind it like this.
Also for a patch release, we should avoid adding new functionality. We could leave a comment or open an issue if we are uncertain.